lib/features/mailbox_creator/domain/extensions/mailbox_name_special_character_validator_extension.dart

```diff
@@ -0,0 +1,15 @@
+
+extension MailboxNameSpecialCharacterValidatorExtension on String {
+  bool get isValid {
+    if (startsWith('#')) {
+      return false;
+    }
+
+    final forbiddenChars = RegExp(r'[%*\r\n]');
```

Comment:
why you have this list of chars?

Reply:
That is from jmap server response in `Mailbox/set`

<img width="931" alt="Screenshot 2024-08-12 at 16 03 54" src="https://github.com/user-attachments/assets/0324d787-5e65-4708-81ae-91c43838cc88">